src/pipeline/TxRxThread.cpp

```diff
@@ -268,11 +268,13 @@ void TxRxThread::initializePipeline_()
         auto txAttenuationStep = new LevelAdjustStep(outputSampleRate_, []() {
             double dbLoss = g_txLevel / 10.0;
             
+#if 0
             if (freedvInterface.getTxMode() == FREEDV_MODE_RADE)
             {
                 // Attenuate by 4 dB as there's no BPF; anything louder distorts the signal
                 dbLoss -= 4.0;
             }
+#endif // 0
             
```

Comment:
The level of the RADE signal should be set at the float to int conversion stage (correctly done above :point_up:), not tweaked later in the Tx signal processing.  This code should be rm-ed, not just if-deffed out.

Reply:
Removed in https://github.com/drowe67/freedv-gui/pull/780/commits/2c3f6a20aaff587739b3ca1577fc4880e3f30820.